src/python/metrics/fuzzer_stats_schema.py

```diff
@@ -0,0 +1,426 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fuzzer stats schema."""
+
+_AFL_SCHEMA = [{
```

Comment:
this is taken as the union of the internal + external schema for AFL. the internal one has some weird columns (e.g. recommended_dict_size) that somehow got in there.

Removing columns is difficult in bigquery, so leaving these here to make sure our load jobs pass, and we can think about how to remove them in the future.

Reply:
Yes we should enforce some order for readability (like stuff such as job/fuzzer/strings first, then top stats, then strategies or something). also remove imisedredundant stuff later.

Reply:
I guess we can define a new schema without all those old/redundant columns, create a new table with the new schema, and migrate the old data into the new table.